src/IB/IBFEMethod.cpp

```diff
@@ -636,6 +636,33 @@ IBFEMethod::interpolateVelocity(const int u_data_idx,
         batch_vec_assembly(U_rhs_vecs);
     }
 
+    // If the velocity system has constraints then those need to be dealt with
+    // here and not during assembly because we do not distribute the
+    // constraints corresponding to the IB partitioning.
+    std::vector<PetscVector<double>*> vecs_for_second_summation;
+    for (unsigned int part = 0; part < d_meshes.size(); ++part)
+    {
+        // We do not need to constrain anything when we have nodal quadrature
+        // since all RHS values are independent
```

Comment:
I don't understand this comment. There are still hanging node values that need to be constrained, regardless of the quadrature scheme. Should this say something like "constraints are dealt with elsewhere"?

Reply:
I need to reword this. My original point was that if we use nodal quadrature then we do not need to use constraints when setting up the linear system since no dof depends on any other dof.

Reply:
I added some more text explaining that FEProjector will always enforce constraints after we solve the linear system.